tests/app/models.py

```diff
@@ -19,7 +19,8 @@ class DataModel(models.Model):
     nullable = models.IntegerField(null=True)
     hasdefault = models.IntegerField(default=1)
     relation = models.ManyToManyField(User)
-    foreign_key = models.ForeignKey(User, null=True, on_delete=models.SET_NULL)
+    mandatory_foreign_key = models.ForeignKey(User, on_delete=models.SET_NULL)
```

Comment:
not that it matters too much in a test context, but having `on_delete=models.SET_NULL` on a non nullable FK feels weird. Maybe set `models.CASCADE` or `models. PROTECT`